src/lib/dom-api.html

```diff
@@ -593,6 +593,61 @@
 
     };
 
+    Object.defineProperties(DomApi.prototype, {
+      activeElement: {
+        configurable: true,
+        get: function() {
+          var ownerDocument = this.node.ownerDocument;
+
+          // Shadow DOM
+          if (Polymer.Settings.useShadow) {
+            if (ownerDocument || !window.ShadowDOMPolyfill) {
+              return this.node.activeElement;
+            } else {
+              return window.ShadowDOMPolyfill.wrapIfNeeded(this.node).activeElement;
+            }
+          }
+
+          // Shady DOM
+
+          var trueDocument = ownerDocument || this.node;
+
+          // Not a document or shady root.
+          if (ownerDocument && !this.node._isShadyRoot) {
+            return this.node.activeElement;
```

Comment:
This is for specifically when it isn't a document or a shady root, it just pulls the property off the node by name. This was the getter portion of the setter I had in earlier that you wanted me to remove, maybe this would be better returning null? I suppose without the setter it's not really helpful.